src/server/HttpResponse.cpp

```diff
@@ -38,41 +40,63 @@ void	HttpResponse::setBody(const std::string bodyContent)
 	_body = bodyContent;
 }
 
-/// @todo  how return valuse used in other functions?
-///			- check Server.cpp
+std::string	HttpResponse::getBody()
+{
+	return (_body);
+}
+std::string HttpResponse::_getResponseLine() const
+{
+	std::ostringstream oss;
+	oss << "HTTP/1.1 " << _statusCode << " " << _statusMessage << "\r\n";
+	return (oss.str());
+}
+
 std::string	HttpResponse::toString() const
 {
-	return (_getHeadersString() + "\r\n\r\n" + _body);
+	std::cout << "TEST | HttpResponse::toString" <<  std::endl;
+	std::cout << _getResponseLine() + _getHeadersString() + "\r\n\r\n" + _body << std::endl;
+	return (_getResponseLine() + _getHeadersString() + "\r\n\r\n" + _body);
 }
 
+std::string	getDefualtPagePath(int page_code)
+{
+	std::ostringstream oss;
+	oss << "./www/error_pages/" << page_code << ".html";
+	return (oss.str());
+}
 
 /// @brief Creates an HttpResponse object by reading the contents of a file.
 /// 
-/// @param filePath The path to the file to be read.
+/// @param file_path The path to the file to be read.
 /// @return HttpResponse The created HttpResponse object.
-HttpResponse	HttpResponse::fromFile(const std::string filePath)
+HttpResponse	HttpResponse::fromFile(const std::string file_path)
```

Comment:
- Suggestion: How about making another constructor for HttpResponse class that take an input parameter of `file_path` to initiate, instead of using a method to make and return an instance of it.

```cpp
HttpResponse::HttpResponse(const std::string file_path)
	: _statusCode(200), _statusMessage("OK")
{
	fromFile(file_path);
}
```